content/docs/apps/continuous-deployment.md

```diff
@@ -52,14 +54,14 @@ $ cf delete-service my-deployer-account
 {{% eastwest %}}
 ### Ask an admin
 
-The first thing you need to do is ask an admin to setup a "deployer" user in your organization and give it permission to deploy to the desired space:
+Ask an admin to setup a "deployer" user in your organization and give it permission to deploy to the desired space.
```

Comment:
Should this be [Ask support](https://cloud.gov/docs/help/) rather than ask an admin?